packages/cdktf-cli/src/lib/dependencies/package-manager.ts

```diff
@@ -9,8 +9,75 @@ import { Errors } from "../errors";
 import * as fs from "fs-extra";
```

Comment:
I think this file has grown to a size where it makes sense to split each package manager into its own file. I would want to do this in a separate PR since I'm not sure which other PRs are potentially touching this and to keep the change set cleaner.